Dockerfile.proto

```diff
@@ -0,0 +1,70 @@
+# syntax=docker/dockerfile:1
+
+ARG UBUNTU_VERSION=22.04
+ARG TFSERVING_VERSION=2.9.0
+
+# Build image, genearates proto files
+FROM arm64v8/ubuntu:${UBUNTU_VERSION}  AS build-image
+
+ARG TFSERVING_VERSION
+
+RUN <<EOF
+apt-get update && apt-get install -y --no-install-recommends \
+python3 \
+python3-pip \
+git \
+&& rm -rf /var/lib/apt/lists/*
```

Comment:
indent for readability - same comment for all RUN <<EOF blocks

Reply:
Done